src/lib/apiTypesV2/Registration.cpp

```diff
@@ -36,7 +36,7 @@ namespace ngsiv2
 *
 * ForwardingInformation::ForwardingInformation -
 */
-ForwardingInformation::ForwardingInformation(): lastFailure(0), lastSuccess(0), timesSent(0), lastForwarding(0)
+ForwardingInformation::ForwardingInformation(): lastFailure(0), lastSuccess(0), timesSent(0), counter(0), maxLimit(0), lastForwarding(0)
```

Comment:
`counter` and `maxLimit` are too broad names. I'd suggest to use some more specific ones, such as:

failsCounter
maxFailsLimit

Reply:
I'd just realized this change is in Registration.cpp file

Note that #3541refers only to subscriptions. Registrations are out of scope. Please revert the change to Registration.cpp and Registration.h in your PR.

Reply:
Fixed in https://github.com/telefonicaid/fiware-orion/pull/3890/commits/49b0c6df2c4d993b4e0fec160026fe0ac0017cb7